rest-api/core-rest/src/main/webapp/lib/gaffer.js

```diff
@@ -135,6 +135,15 @@ function init(onSwaggerComplete){
           log("Loaded swagger");
               $('pre code').each(function(i,e){hljs.highlightBlock(e)});
               addExampleButtons();
+              $.get(
+                    getVersion() + '/graph/operations/uk.gov.gchq.gaffer.operation.impl.job.GetJobDetails',
+                    null,
+                    function(response) {
+                        if(200 != response.response_code) {
```

Comment:
This callback will only run on success, not sure you really need the check.  If you want it is possible to use function(responseData, statusText) {if(statusText !== "Success")()}.  Better way to check the response.